Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -73,15 +88,24 @@ public static boolean getBooleanParameter(String parameter, boolean defaultIfNul
     }
 
     /**
-     * Request selected parameter with given default value from configuration.
+     * Request int-parameter from Configuration.
      *
-     * @return Parameter as String
+     * @return Parameter as Int
      */
-    public static String getParameter(String parameter, String defaultIfNull) {
+    public static int getIntParameter(String inParameter) {
+        return getIntParameter(inParameter, 0);
+    }
+
+    /**
+     * Request int-parameter from Configuration with default-value.
+     *
+     * @return Parameter as Int
+     */
+    public static int getIntParameter(String inParameter, int inDefault) {
         try {
-            return getConfig().getString(parameter, defaultIfNull);
+            return getConfig().getInt(inParameter, inDefault);
         } catch (RuntimeException e) {
-            return defaultIfNull;
+            return 0;
```

Comment:
Why are you returning `0` instead of value from `inDefault` like in other methods?